snappy/kernel.go

```diff
@@ -38,6 +38,9 @@ func dropVersionSuffix(name string) string {
 	return strings.SplitN(name, "-", 2)[0]
 }
 
+// used in the unittests
+var bootloader = partition.BootloaderDir
```

Comment:
Thanks Gustavo, good catch! This was indeed unused. sorry for this. The right one is bootloaderDir() that was actually defined in systemimage.go. Because systemimage.go will go away soon(ish) I moved this now to kernel.go.